.github/workflows/python-package-conda.yml

```diff
@@ -0,0 +1,34 @@
+name: Python Package using Conda
```

Comment:
I think this part of the PR is out of scope. Furthermore, testing a conda build is already done in the Biotite feedstock in conda-forge. I am in favor of flake8 formatting but to me it is another topic, especially I would prefer to keep it strict and require proper flake8 formatting in all source files. Hence, this would require some effort to fix the flake8 findings in all source files.